### DISCUSSION TITLE
Callout - show in TOC

### DISCUSSION THREAD

**Discussion by stefansimik** [quarto-dev/quarto-cli#3732] [Q&A] [answered]
I would like to suggest, that content of the callouts should be optionally displayed in TOC.
Currently, it is completely ignored.

I mean block like this:

```
::: {.callout-note appearance="simple"}

## Additional info

Using callouts is an effective way to highlight content that your reader give special consideration or attention.

:::
```

should be optionally displayed in TOC as  clickable link with text **Additional info**.
This would be really useful, if the content is important part of the document, which should be referenced in TOC. 
It could be simply by implemented by adding new parameter `toc: true/false`, with default value false for backwards compatibility.

What do you think about it guys?

**Comment by mcanouil** [comment, marked as answer]
In callouts, the level 2 header does not have the usual level 2 header meaning, thus how Quarto can tell at which level it is supposed to insert a reference/link?

In my opinion, callouts are by definition paragraphs with special emphasis such as theorems, etc., from that point, I can't see why a paragraph should be part of a table of content.

**Comment by stefansimik** [comment]
Thanx, makes sense
